### ISSUE TITLE
High price orders are not working

### ISSUE THREAD

**Issue by cmagan** [gnosis/ido-ux#262]
When trying to submit order with high price, the spinning wheel in the confirmation modal keeps spinning but no metamask pop up appears.
![image.png](https://images.zenhubusercontent.com/60227f03bd43d8095c1185d0/ade4c639-f479-44f4-8395-01c8aee1ccb6)

**Comment by mariano-aguero** [comment]
The _buyAmountScaled_ calculated in the method _convertPriceIntoBuyAndSellAmount_ was zero, so the contract will trigger a revert exception. We are using the BigNumber library from ethers, maybe we need to use another library to calculate with more precision the value to use (for example Big.js)

**Comment by josojo** [comment]
>The buyAmountScaled calculated in the method convertPriceIntoBuyAndSellAmount was zero, so the contract will trigger a revert exception. We are using the BigNumber library from ethers, maybe we need to use another library to calculate with more precision the value to use (for example Big.js)

Yeah, i think the restriction is within the smart contract. It can't deal with these numbers. I think a simple warning in UI that the price is too high and a user should choose a lower price is sufficient.

Sorry that I have not seen this earlier.

**Comment by elena-zh** [comment]
Issues reported in this comment https://github.com/gnosis/ido-ux/issues/262#issuecomment-804728549 are fixed. 
The main issue in the description is still reproducible.
